rollbar-client/package.yaml

```diff
@@ -38,7 +38,7 @@ _exe-ghc-options: &exe-ghc-options
 library:
   source-dirs: src
   dependencies:
-    - aeson >= 1.4 && < 2
+    - aeson >= 2.0 && <= 2.2
```

Comment:
@cptrodolfox I think we could be more flexible on the upper bound and allow changes (< 3.0)

Reply:
@sestrella Thanks for the suggestion I will be updating the upper bound.